lib/middleware.ts

```diff
@@ -86,6 +103,14 @@ var defaultHandlers = [
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  () => ({source: "kayvee"}),
```

Comment:
If they are only left for testing, then we should probably not allow them to be passed in from the exported module at all by default. If we want to keep them for (2) then we should consider it supported. My personal preference for now is to start with a strong interface that does not allow modifying morgan outside of kayvee